README.md

```diff
@@ -76,10 +76,22 @@ cf map-route <APP_NAME> <DOMAIN> --hostname <HOSTNAME>
     Find out more about [configuring your CI tool accounts](https://docs.cloud.service.gov.uk/using_ci.html#configure-your-ci-tool-accounts) in GOV.UK PaaS.
 9. [Store the newly created credentials in GitHub Actions][store_creds] - You should store your sensitive credentials in GitHub Actions. 
     Store the username with secret name `GOV_PAAS_USER` and the password with secret name `GOV_PAAS_PASS`.
-10. OPTIONAL: Create a shared username/password for accessing the hosted dashboard.
-    This can be useful if you want to prevent curious individuals from accessing your dashboard while in development, but does not give any security against malicious actors.
-    You will need to have access to the `cf` command installed and configured, which currently isn't the case within the DAP.
-    GOV.UK PaaS provide guidance on how to do this under the title [Example: Route service to add username and password authentication][basic_auth].
+
+## OPTIONAL: Create a shared username/password for accessing the hosted dashboard
+This can be useful if you want to prevent curious individuals from accessing your dashboard while in development, but does not give any security against malicious actors.
+You will need to have access to the `cf` command installed and configured, which can be requested through DAP support.
+
+GOV.UK PaaS provide guidance on how to do this under the title [Example: Route service to add username and password authentication][basic_auth], however following this method you cannot have both basic authentication and IP restricitons in place as your app can only have one routing service. If you would like your app to have basic authentication and IP restricitons, please follow the below steps to add basic authentication using flask:
+
+1. In `bootstrap.py` uncomment the `enable_basic_auth` function
```

Comment:
Won't the `enable_basic_auth` method need to be called by something?  I'm not confident uncommenting by itself would be good enough?

I'm wondering if it makes sense to add this method to our shared library if it's going to be shared across multiple dashboards?